primitives/src/errors.rs

```diff
@@ -42,26 +32,17 @@ pub enum PrimitivesError {
     InconsistentStructureError(String),
```

Comment:
nit: Are these still used/relevant after `MerkleTreeError` has been introduced?

Reply:
removed in [ac0c37b](https://github.com/EspressoSystems/jellyfish/pull/556/commits/ac0c37bcb8daaef34ea55be9e6456d78f8359216)